lib/tml/cache_adapters/memcache.rb

```diff
@@ -93,7 +93,7 @@ def exist?(key, opts = {})
     data = @cache.get(versioned_key(key, opts))
     not data.nil?
   rescue Exception => ex
-    warn("Failed to check if key exists: #{key}")
+    warn("Failed to check if key exist: #{key}")
```

Comment:
Similar to the comment changes, I think we are okay keeping the original wording of "exists" instead of "exist"